istioctl/cmd/root.go

```diff
@@ -299,3 +299,15 @@ func seeExperimentalCmd(name string) *cobra.Command {
 		},
 	}
 }
+
+// setNotSupportedCmd is used to set commands that are not supported for OpenShift Service Mesh.
+func setNotSupportedCmd(name string) *cobra.Command {
+	msg := fmt.Sprintf("Command not supported. `%s` cannot be used in OpenShift Service Mesh context.", name)
```

Comment:
I think we need to explain why it's unsupported and what the alternative is (for each command). I hate it when e.g. a function is deprecated but it doesn't tell you what to use instead.

Reply:
Is adding this to the doc enough?

Reply:
Done